extensions/logging/src/keys.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/extension-logging
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {BindingKey, GenericInterceptor} from '@loopback/core';
+import {FluentSender} from 'fluent-logger';
+import {Logger} from 'winston';
+import * as Transport from 'winston-transport';
+import {LoggingComponent} from './logging.component';
+
+/**
+ * Binding keys used by this component.
+ */
+export namespace LoggingBindings {
+  export const COMPONENT = BindingKey.create<LoggingComponent>(
+    'components.LoggingComponent',
+  );
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  export const FLUENT_SENDER = BindingKey.create<FluentSender<any>>(
+    'logging.fluent.sender',
+  );
+
+  export const WINSTON_LOGGER = BindingKey.create<Logger>(
+    'logging.winston.logger',
+  );
```

Comment:
IIUC, we agreed to build the logging extension around Winston module, making the logger interface hard-coded. I find it redundant to include `WINSTON_` prefix in the keys, because there aren't going to be non-winston keys. We already have `LoggingBindings` as a namespace.

Proposed constant: `LOGGER` and key value: `logging.logger`.

(Same comment applies to other keys below).

Reply:
I intentionally have `winston` in the names to remind our users to follow winston interfaces/types.